Mathlib/RingTheory/Derivation/MapCoeffs.lean

```diff
@@ -0,0 +1,98 @@
+/-
+Copyright (c) 2024 Daniel Weber. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Daniel Weber
+-/
+import Mathlib.RingTheory.Derivation.Basic
+import Mathlib.Algebra.Polynomial.Module.Basic
+
+/-!
+# Derivations of univariate polynomials
+
+In this file we prove properties about `R`-derivations of `Polynomial A`.
```

Comment:
I feel like this is a bit of a blank statement. Eg you are not proving anything about the obvious derivation on polynomials which sends `X ^ n` to `n • X ^ (n - 1)`. Can you expand?